go.mod

```diff
@@ -11,20 +11,20 @@ replace (
 )
 
 require (
-	github.com/ajeddeloh/go-json v0.0.0-20170920214419-6a2fe990e083 // indirect
-	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
-	github.com/coreos/ignition v0.33.0
+	github.com/coreos/ign-converter v0.0.0-20200825151652-ea20012f9844
+	github.com/coreos/ignition v0.35.0
+	github.com/coreos/ignition/v2 v2.6.0
 	github.com/go-bindata/go-bindata/v3 v3.1.3
 	github.com/go-logr/zapr v0.1.0
 	github.com/gogo/protobuf v1.2.1 // indirect
+	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.3
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.5.1
 	github.com/vincent-petithory/dataurl v0.0.0-20160330182126-9a301d65acbb
 	go.uber.org/atomic v1.4.0 // indirect
 	go.uber.org/zap v1.10.0
-	go4.org v0.0.0-20190919214946-0cfe6e5be80f // indirect
-	golang.org/x/sys v0.0.0-20190403152447-81d4e9dc473e
+	golang.org/x/sys v0.0.0-20200610111108-226ff32320da
```

Comment:
are these updates required as a part of this change?

Reply:
They aren't pinned via a replace directive, so `go mod tidy` has updated them here to a version it deems compatible.